src/_includes/get-sdk.md

```diff
@@ -40,9 +37,12 @@ You'll need it in the next step.
   installing Dart is easy.
 
   ```terminal
-  > brew tap dart-lang/dart
-  > brew install dart
+  $ brew tap dart-lang/dart
+  $ brew install dart
   ```
 </div>
 
-More information: [Get the Dart SDK](/get-dart)
+{{site.alert.important}}
+  For more information, including how to **adjust your `PATH`**, see
+  [Get the Dart SDK](/get-dart).
+{{site.alert.end}}
```

Comment:
I added this alert to address the following need that used to be documented in src/_tutorials/server/get-started.md: "PENDING: the following instructions assume you have set the PATH. We should update the included instructions to refer to that."